etcd/map.jinja

```diff
@@ -1,6 +1,6 @@
 {% set server = salt['grains.filter_by']({
     'Debian': {
-        'pkgs': ['etcd', 'python-etcd'],
```

Comment:
In this case I would like to have following options:
- pkg (install using pkg as it's done now)
- docker (run executing docker, similar to aptly)
- docker-binary (some hybrid like this one)